View Controllers/TimelineViewController.m

```diff
@@ -108,8 +110,13 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     cell.usernameHandleLabel.text = [@"@" stringByAppendingString: tweet.user.screenName];
     
     //timestamp/date published
-    cell.timestampLabel.text = tweet.createdAtString;
+    NSString *timestampString = tweet.createdAtString;
+    NSDateFormatter *format = [[NSDateFormatter alloc] init];
```

Comment:
Date formatter creation is notoriously expensive. I remember reading somewhere that setting the date format might be the actually expensive part but either way this should go in a constant outside of this method.